app/waf/src/controllers/member.controller.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Member} from '../models';
+import {MemberRepository} from '../repositories';
+import uuid = require('uuid');
+
+const prefix = '/adcaas/v1';
+
+export class MemberController {
+  constructor(
+    @repository(MemberRepository)
+    public memberRepository: MemberRepository,
+  ) {}
+
+  @post(prefix + '/members', {
+    responses: {
+      '200': {
+        description: 'Member model instance',
+        content: {'application/json': {schema: {'x-ts-type': Member}}},
+      },
+    },
+  })
+  async create(@requestBody() member: Member): Promise<Member> {
```

Comment:
As mentioned f2f, we need to chagne `Member` to `Partial<Member>` to allow user to input part of Member instance.